src/_DetailBase.js

```diff
@@ -567,9 +638,31 @@ const __class = declare('argos._DetailBase', [View], {
 
       if (!sectionStarted) {
         sectionStarted = true;
-        const section = domConstruct.toDom(this.sectionBeginTemplate.apply(layout, this) + this.sectionEndTemplate.apply(layout, this));
-        sectionNode = section.childNodes[1];
-        domConstruct.place(section, this.contentNode);
+        if (this.isTabbed) {
+          if (layout.name === 'QuickActionsSection') {
```

Comment:
Weren't we going to add another configuration option to check so we won't have to rely on the magic value of "QuickActionsSection"?